Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -316,11 +319,13 @@ void MainWindow::Pause()
 
 bool MainWindow::Stop()
 {
-  if (!Core::IsRunning())
+  if (!Core::IsRunning() || m_inquiry_in_progress)
```

Comment:
I don't understand this either. Isn't the confirm dialog already modal? How do you trigger multiple dialog boxes?